TypeDefinition/AdditiveManufacturing/Material.hpp

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2022 (c) Patrick Moeller, 3Yourmind GmbH
+ */
+
+#pragma once
+#include <string>
+
+#include "../../OpcUaTypes/EUInformation.hpp"
+#include "../../UmatiServerLib/BindableMemberValue.hpp"
+#include "../TypeDefinition.hpp"
+#include "../ns0/AnalogUnitType.hpp"
+#include "../ns0/Constants.hpp"
+#include "Constants.hpp"
+
+namespace AdditiveManufacturing {
+
+struct Material_t {
+  BindableMemberValue<bool> ReadyForProduction;
+  BindableMemberValue<std::string> ExternalIdentifier;
+  BindableMemberValue<std::string> Identifier;
+  BindableMemberValue<std::string> Function;  // FIXME: Add enum MaterialFunction
```

Comment:
See https://github.com/umati/Sample-Server/blob/5b0082cd0e950fb36755ebe9989121531c23388b/TypeDefinition/IA/StackElementLight.hpp#L20 on how to use Enums.